static/src/javascripts/projects/commercial/modules/third-party-tags.js

```diff
@@ -12,28 +12,38 @@ import { ias } from 'commercial/modules/third-party-tags/ias';
 import { inizio } from 'commercial/modules/third-party-tags/inizio';
 import { fbPixel } from 'commercial/modules/third-party-tags/facebook-pixel';
 import { init as initPlistaOutbrainRenderer } from 'commercial/modules/third-party-tags/plista-outbrain-renderer';
+import { onConsentNotification } from 'lib/cmp';
+
+let scriptsInserted: boolean = false;
 
 const insertScripts = (services: Array<ThirdPartyTag>): void => {
-    const ref = document.scripts[0];
-    const frag = document.createDocumentFragment();
-    let insertedScripts = false;
+    onConsentNotification('advertisement', state => {
+        if (!scriptsInserted && (state === true || state === null)) {
+            scriptsInserted = true;
 
-    services.forEach(service => {
-        // flowlint sketchy-null-bool:warn
-        if (service.useImage) {
-            new Image().src = service.url;
-        } else {
-            insertedScripts = true;
-            const script = document.createElement('script');
-            script.src = service.url;
-            script.onload = service.onLoad;
-            frag.appendChild(script);
-        }
-    });
+            const ref = document.scripts[0];
+            const frag = document.createDocumentFragment();
+            let hasScriptsToInsert = false;
+
+            services.forEach(service => {
+                if (service.useImage === true) {
```

Comment:
can useImage be null? if not we can remove === true

Reply:
It cannot be null but it can be undefined. The previous version didn't have the `=== true` but was throwing an error so I changed it to keep the linter happy.